### DISCUSSION TITLE
Lowering the tone of a song in the editor

### DISCUSSION THREAD

**Discussion by Naabway1** [ppy/osu#27830] [Q&A] [answered]
### Type

Other

### Bug description

Hi peppy, I'm an std mapper and have been trying to move to laser for a long time, but I'm stressed by the lowering of the song tone at the moment you slow the song down, as if reverb is superimposed. Maybe it's a feature, but it's very unusual for me and discourages me as a mapper, if it's a feature, can you add a checkbox to the settings on the editor, thanks.

### Version

2024.312.1

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14950067/compressed-logs.zip)
but I guess it doesn't really matter.

**Comment by Walavouchey** [comment, marked as answer]
there is no "reverb". the pitch changes when slowing down the song in order to prevent audio distortion, and audio distortion makes timing inaccurate
